handlers/handlers.go

```diff
@@ -331,7 +331,7 @@ func filterableLanding(w http.ResponseWriter, req *http.Request, dc DatasetClien
 	}
 
 	if cfg.EnableCensusPages && strings.Contains(datasetModel.Type, "cantabular") {
-		censusLanding(ctx, w, req, dc, datasetModel, rend, edition, ver, displayOtherVersionsLink, allVers, collectionID, lang, userAccessToken)
+		censusLanding(ctx, w, req, dc, datasetModel, rend, edition, ver, displayOtherVersionsLink, allVers, latestVersionNumber, latestVersionOfEditionURL, collectionID, lang, userAccessToken)
```

Comment:
Just making sure... this is supposed to be `latestVersionOfEditionURL` and not `latestVersionURL`?

Reply:
Yep, I am just reusing some existing code. Probably could do with a rename as the helper that's used to generate it on line 319 is called `DatasetVersionUrl`. I'll rename it and push the change